esp/src/eclwatch/ActivityWidget.js

```diff
@@ -583,6 +586,15 @@ define([
             this.wuMoveUpButton.set("disabled", !wuCanUp);
             this.wuMoveDownButton.set("disabled", !wuCanDown);
             this.wuMoveBottomButton.set("disabled", !wuCanDown);
+        },
+
+        createStackControllerTooltip: function (widgetID, text) {
+            return new Tooltip({
+                connectId: [widgetID],
+                label: text,
+                showDelay: 1,
+                position: ["below"]
```

Comment:
I am using tooltip position similar to the main icons (bottom). We do however use a "title" type attribute in other tooltips such as the ones on tabs but, they do not have the same appearance.